### PR TITLE
Update pydata-sphinx-theme to allow update and fix crash in sphinx docs build

### PR DESCRIPTION
**Pull request by germa89**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -40,9 +40,7 @@ def read(rel_path):
     include_package_data=True,
     # See http://www.sphinx-doc.org/en/stable/theming.html#distribute-your-theme-as-a-python-package
     entry_points={"sphinx.html_themes": ["pyansys_sphinx_theme = pyansys_sphinx_theme"]},
-    install_requires=["sphinx",
-                      "pydata-sphinx-theme==0.6.3",
```
Comment (akaszynski) [inline setup.py:44]: You still need the theme.

Comment (germa89) [reply setup.py:44]: I added it to the requirements now. I realized later.

**Comment by germa89** [comment]
@akaszynski I guess now.... ?

**Comment by akaszynski** [comment]
No, because the theme install requires must have the pydata theme.  You should just upgrade the install requires.  See suggestions.

File: requirements_docs.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 sphinx==4.0.3
+pydata_sphinx_theme
```
Comment (akaszynski) [inline requirements_docs.txt:2]:
```suggestion
sphinx
```

**Comment by akaszynski** [comment]
We no longer need install requirements_docs, right?

**Comment by germa89** [comment]
True

**Comment by akaszynski** [comment]
You need to change the workflow in .github/workflows

**Review by akaszynski — APPROVED** [review]
LGTM

File: setup.py [hunk 3]
```diff
@@ -40,9 +40,7 @@ def read(rel_path):
...
-                      ],
+    install_requires=["sphinx"],
```
[the remark this answers is not in the document]
Comment (germa89) [reply setup.py]: I guess better not fix the version ?

[the remark this answers is not in the document]
Comment (akaszynski) [reply setup.py]: It's better to fix the version because if we don't incorporate upstream changes, we run the risk of breaking something.

**Pull request merged**